src/domain/proposal.ts

```diff
@@ -65,6 +65,16 @@ export function getProposalIPFSData(proposal: Proposal): Proposal {
         if (descJson.toObject().get('url') != null) {
           proposal.url = descJson.toObject().get('url').toString();
         }
+        let tags: string[] = [];
+        if (descJson.toObject().get('tags') != null && descJson.toObject().get('tags').kind === JSONValueKind.ARRAY) {
+          let tagsObjects = descJson.toObject().get('tags').toArray();
+          for (let i = 0; i < tagsObjects.length; i++) {
```

Comment:
does the length should be capped somehow by max number of tags per proposal ? other wise this code might be fragile .

Reply:
@jellegerbrandy does capping that with 100 tags per proposals sound reasonable ?

Reply:
@ben-kaufman please cap the number of tags per proposal with the magical number 100